scripts/do-all-unix.sh

```diff
@@ -24,20 +24,16 @@ fi
 
 DESTINATION=$(pwd)/destination
 
-# set BUILD_STATIC_LIB and BUILD_SHARED_LIB depending on LIBRARY_TYPE
-BUILD_STATIC_LIB=OFF
-BUILD_SHARED_LIB=OFF
-if [ "$LIBRARY_TYPE" == "SHARED" ]; then
-  BUILD_SHARED_LIB=ON;
-elif [ "$LIBRARY_TYPE" == "STATIC" ]; then
-  BUILD_STATIC_LIB=ON;
+# set BUILD_SHARED_LIBS depending on LIBRARY_TYPE
+BUILD_SHARED_LIBS=ON
+if [ "$LIBRARY_TYPE" == "STATIC" ]; then
+  BUILD_SHARED_LIBS=OFF;
 fi
 
 ./scripts/build-unix.sh                      \
     -DCMAKE_BUILD_TYPE=$BUILD_TYPE           \
     -DCMAKE_INSTALL_PREFIX=$DESTINATION      \
-    -DBUILD_STATIC_LIB=$BUILD_STATIC_LIB     \
-    -DBUILD_SHARED_LIB=$BUILD_SHARED_LIB     \
+    -DBUILD_SHARED_LIBS=$BUILD_SHARED_LIBS     \
```

Comment:
very small nitpicking
```suggestion
    -DBUILD_SHARED_LIBS=$BUILD_SHARED_LIBS   \
```